src/App.jsx

```diff
@@ -1,9 +1,150 @@
+                    <span className="image-product">
+                        <img src={bestSellingTv.sourceImg} alt="Image product"/>
+                    </span>
+                    <article className="article-box-2">
+                        <h3>{collectProductName(bestSellingTv)}</h3>
+                        <h2>{collectProductPrice(bestSellingTv)}</h2>
+                        <h4>{collectConvertScreenSizes(bestSellingTv)}</h4>
+                        <ul className="productOptions">
+                            <li><img className="iconSize" src={check} alt="Wifi als optie aanwezig" />wifi</li>
+                            <li><img className="iconSize" src={minus} alt="Speech niet als optie aanwezig" />speech</li>
+                            <li><img className="iconSize" src={check} alt="HDR als optie aanwezig" />hdr</li>
+                            <li><img className="iconSize" src={check} alt="Bluetooth als optie aanwezig" />bluetooth</li>
+                            <li><img className="iconSize" src={minus} alt="ambilight niet als optie aanwezig" />ambilight</li>
+                        </ul>
+
+                    </article>
+                </div>
+            </section>
+
+            <section>
+                <h2>Alle TV's</h2>
+                <div className="sortButtons">
+                    <button type="button" onClick={sortedByBestSeller}>Meest verkocht eerst</button>
+                    <button type="button" onClick={sortedByCheapestTv}>Goedkoopste eerst</button>
+                    <button type="button" onClick={sortedByBestSportsTv}>Meest geschikt voor sport eerst</button>
+                    {/*DEEL 2 - BONUS 1 */}
+                    <button type="button" onClick={sortedByScreenSize}>Grootste schermgroottes eerst</button>
+                </div>
+            </section>
+
+            <section className="brandOverview">
+                <h3 className="allignLeft">Overzicht van onze beschikbare merken</h3>
+                <ul>
+                    {inventory.map((item) => <li key={`key-${item.type}`}>{item.brand}</li>)}
+                </ul>
+            </section>
+
+            <section>
+                {inventory.map((item) => {
+                    return (
+                        <div className="article-container-3" key={`key-${item.type}`}>
+                            <span className="image-product">
+                                <img src={item.sourceImg} alt="image-tv"/>
+                            </span>
+                            <article className="article-box-3">
+                                <h3>{collectProductName(item)}</h3>
+                                <h2>{collectProductPrice(item)}</h2>
+                                <h4>{collectConvertScreenSizes(item)}</h4>
+                                <ul className="productOptions">
+                                    {item.options.map((feature) => {
+                                        if (feature.applicable === true) {
```

Comment:
Dit zou je nog kunnen verkorten
```suggestion
                                        if (feature.applicable) {
```